### PR TITLE
fix issue #11: add initial support for Markdown

### PR DESCRIPTION
**Pull request by Qiwen-Yu**
Update and comment code to support the level 1 heading parsing from Mardown file to html file.

### CODE REVIEW

**Comment by Qiwen-Yu** [comment]
The corresponding part of README.md file was also got updated.

**Comment by humphd** [comment]
@roman-rezinkin it looks like @Qiwen-Yu's PR is ready for review, can you please provide some feedback and merge when ready?

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (roman-rezinkin) [inline .idea/.gitignore:1]: Did you mean to add all the .idea folders? I use JetBrains products and it auto generates them. I am just wondering if it is essential to add these to the code.

File: .idea/RomanStaticSG.iml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (roman-rezinkin) [inline .idea/RomanStaticSG.iml:1]: Did you mean to add all the .idea folders? I use JetBrains products and it auto generates them. I am just wondering if it is essential to add these to the code.

File: .idea/inspectionProfiles/Project_Default.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<component name="InspectionProjectProfileManager">
```
Comment (roman-rezinkin) [inline .idea/inspectionProfiles/Project_Default.xml:1]: Did you mean to add all the .idea folders? I use JetBrains products and it auto generates them. I am just wondering if it is essential to add these to the code.

File: .idea/inspectionProfiles/profiles_settings.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (roman-rezinkin) [inline .idea/inspectionProfiles/profiles_settings.xml:1]: Did you mean to add all the .idea folders? I use JetBrains products and it auto generates them. I am just wondering if it is essential to add these to the code.

File: .idea/misc.xml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (roman-rezinkin) [inline .idea/misc.xml:1]: Did you mean to add all the .idea folders? I use JetBrains products and it auto generates them. I am just wondering if it is essential to add these to the code.

File: .idea/modules.xml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (roman-rezinkin) [inline .idea/modules.xml:1]: Did you mean to add all the .idea folders? I use JetBrains products and it auto generates them. I am just wondering if it is essential to add these to the code.

File: .idea/vcs.xml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (roman-rezinkin) [inline .idea/vcs.xml:1]: Did you mean to add all the .idea folders? I use JetBrains products and it auto generates them. I am just wondering if it is essential to add these to the code.

**Review by roman-rezinkin — CHANGES_REQUESTED** [review]

**Comment by humphd** [comment]
If you decide to remove them:

```
git rm -fr .idea
git commit -m "Remove .idea/"
git push origin issue11
```

**Pull request merged**